CONFIG.md

```diff
@@ -1,5 +1,13 @@
 ```js
 {
+  "maxReviewers": 5, // Maximum  number of people to ping in the PR message
+  "numFilesToCheck": 10, // Number of files to check against, deafault is 5
```

Comment:
nit: default is misspelled.

Also, perhaps include the default value for maxReviewers as well?